lib/parser.rb

```diff
@@ -0,0 +1,15 @@
+class Parser
+  attr_accessor :file_pathname
+
+  def initialize(file_pathname)
+    @file_pathname = file_pathname
+  end
+
+  def get_first_line
+    if File.file?(@file_pathname)
+      first_line = File.foreach(@file_pathname).first(1).join(', ')
```

Comment:
Eu não entendi muito bem o motivo de fazer dessa forma usando o join (pelo que eu vi o `.first(1)` retorna um array com o primeiro elemento), mas daria pra simplificar só fazendo um:
```suggestion
      first_line = File.foreach(@file_pathname).first
```

Já iria retornar a primeira linha e fechar o arquivo automaticamente.